client/packages/programs/src/JsonForms/common/components/Autocomplete.tsx

```diff
@@ -100,7 +103,7 @@ const UIComponent = (props: ControlProps) => {
   return (
     <DetailInputWithLabelRow
       sx={DefaultFormRowSx}
-      label={label}
+      label={t(label as LocaleKey)}
```

Comment:
Added this for all `common` JSON form components, and the couple others we use from default schema. Others are only used from custom schema, so I think fine to use label directly?

This translation is safe, if "label" is not an existing translation key, it will just be rendered as is

Hoped there might be a way to apply some kind of middleware to all labels, before they are passed into each component, so as not to have to do this everywhere. I could only find way to apply middleware to whole schema though, not individual elements 😢